### DISCUSSION TITLE
how to scroll element ,thanks

### DISCUSSION THREAD

**Discussion by techflag** [AutomaApp/automa#330] [Q&A]
how to scroll a data list，eg：https://new.qq.com/tag/82542

or

![image](https://user-images.githubusercontent.com/939362/153624573-71f434bd-096f-4a94-89dc-fafac7d3fc89.png)

tks

**Comment by melikst** [comment]
In your example link, you can choose one of the page bottom elements as **CSS selector** for **Scroll Element Block**: `.footernew` or `#load-more`
![1](https://user-images.githubusercontent.com/45320488/153769148-0fdd1ee6-0c51-460d-a43c-6185fc3a4df3.jpg)
